### PR TITLE
Modify grammatical errors

### PR DESCRIPTION
**Pull request by awesome-doge**
* Remove redundant 's'
* Add a comma
* Change the typo

I hope this will help.

### CODE REVIEW

File: ch01.asciidoc [hunk 1]
```diff
@@ -32,35 +32,35 @@ Due to the used onion routing which is also used in tor even the nodes that are
...
 We'll be seeing them throughout the book:
 
 consumer::
-A regular consumer on the Internet or in the offline world who wants to do purchases
+A regular consumer on the Internet or in the offline world who wants to make purchases
 
 content creator / curator::
 A person or platform offering content on the web.
-They want to install a paywall or get tipped by their fans and consumers.
+They want to install a payroll or get tipped by their fans and consumers.
```
Comment (renepickhardt) [inline ch01.asciidoc]: It is supposed to be pay wall and not payroll

**Review by renepickhardt — CHANGES_REQUESTED** [review]
One of the suggestions seems to be wrong. Could you please fix that? Also could you next time provide such a patch in 1 commit to save space in the repo? Other than that thanks for your help!

File: ch01.asciidoc (hunk 1 above)
Comment (awesome-doge) [reply ch01.asciidoc]: It seems that I have successfully modified it.
Sorry, I am not very familiar with git, I am still studying hard.

**Review by renepickhardt — APPROVED** [review]
Thanks!

**Pull request merged**